packages/components/src/styles/components/app-header.scss

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APP-HEADER
+//
+
+@use "../mixins/interactive-dark-theme" as *;
+
+// ----------------
+// Q: Create tokens?
+$app-header-home-link-padding: 4px;
+
+// Breakpoints WIP: START ----------------------------
+
+// Break points
+$tablet-min-width: 768px;
+$desktop-min-width: 1024px;
+
+// base styles
+.hds-app-header__menu {
+  display: inherit; // may be over-riddden by queries
+  gap: inherit;
+}
+
+// Media queries
+// -------------
+.hds-app-header {
+  // Mobile/small and tablet/medium compound view: applies at tablet min-width and below
+  @media (max-width: #{$desktop-min-width - 1}) {
+    &:not(.hds-app-header__menu-is-open) {
+      .hds-app-header__menu {
+        // dropdown menu is hidden when isOpen = false
+        display: none;
+      }
+    }
+
+    // Display content as dropdown menu in mobile/tablet view
+    .hds-app-header__menu {
+      position: absolute;
+      top: calc(100% + 1px); // We add 1px to account for the bottom border
+      right: 0;
+      left: 0;
+      z-index: 5; // TODO: determine correct z-index to use
+      display: flex;
+      flex-wrap: wrap;
+      // height: calc(100vh - 52px); // it should take the full height of the viewport minus the header height
```

Comment:
Is this commented out specifically for the showcase?

Reply:
Yes, just for now. I plan to add it back and test within HCP & Terraform. I'll add another style to override it within the Showcase.

Reply:
Uncommented height and added additional related layout styles with overrides for Showcase examples.